### DISCUSSION TITLE
Email is not sending on mobile

### DISCUSSION THREAD

**Discussion by jricardocv** [emailjs-com/help#157] [Q&A] [answered]
It is my first time using EmailJS on an web app and I noticed that when using the mobile version of the website, the email is not being sent, even though it works properly when using the desktop version.

**Comment by xr0master** [comment, marked as answer]
The service doesn't care where the code is executed.
I think you clear the form before the request is processed, a common issue for some developers.

**Comment by jricardocv** [comment]
That was the issue, thanks
